src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const count = 40;
+  const discount = count * days;
+
+  if (days >= 3 && days <= 6) {
+    return discount - 20;
```

Comment:
Hardcoded values '3', '6', and '20' should be replaced with constants to make the code more readable and maintainable, as per the checklist item 2.